test/WebJobs.Script.Tests/UtilityTests.cs

```diff
@@ -865,7 +868,78 @@ public void VerifyWorkerIndexingDecisionLogic(bool workerIndexingFeatureFlag, bo
                 testEnv.SetEnvironmentVariable(EnvironmentSettingNames.AzureWebJobsFeatureFlags, ScriptConstants.FeatureFlagEnableWorkerIndexing);
             }
             RpcWorkerConfig workerConfig = new RpcWorkerConfig() { Description = TestHelpers.GetTestWorkerDescription("python", "none", workerIndexingConfigProperty) };
-            bool workerShouldIndex = Utility.CanWorkerIndex(new List<RpcWorkerConfig>() { workerConfig }, testEnv);
+            bool workerShouldIndex = Utility.CanWorkerIndex(new List<RpcWorkerConfig>() { workerConfig }, testEnv, new FunctionsHostingConfigOptions());
+            Assert.Equal(expected, workerShouldIndex);
+        }
+
+        [Theory]
+        [InlineData(true, true, false, "", true)]
+        [InlineData(true, true, true, "NonApp", true)]
+        [InlineData(true, true, true, "AppName", true)]
+        [InlineData(true, true, false, "NonApp", true)]
+        [InlineData(true, true, false, "AppName", true)]
```

Comment:
For this test, is there no scenario where we can get expected output as false?

Reply:
We do,  i have separated the test cases titles so it helps understand what is broken.